DopplerCustomDomain/Controllers/CustomDomainController.cs

```diff
@@ -26,5 +27,26 @@ public class CustomDomainController
             _customDomainProviderService = customDomainProviderService;
             _serviceNameResolver = serviceNameResolver;
         }
+
+        [HttpPost("/{service}/{domain}")]
```

Comment:
What do you think about 

```suggestion
        [HttpPost("/{domain}/{service}")]
```

Reply:
BTW, what do you think about adding the DELETE? (the backend is already done)